packages/compass-web/polyfills/net/index.ts

```diff
@@ -8,13 +8,7 @@ import { Duplex } from 'stream';
  * implementation
  */
 class Socket extends Duplex {
-  private _tls = false;
   private _ws: WebSocket | null = null;
-  private _setOptions: {
```

Comment:
Drive-by clean-up, this was doing nothing after a recent change to the logic of the proxy